charts/argo-cd/templates/argocd-server/service.yaml

```diff
@@ -1,12 +1,12 @@
 apiVersion: v1
 kind: Service
 metadata:
-{{- if .Values.server.service.annotations }}
   annotations:
+  {{- if .Values.server.service.annotations }}
```

Comment:
Why move this below the key? w/out default values in the annotations you get an empty object. Not that it should matter here, but could cause confusion debugging chart in the future